tests/test_symbolics.py

```diff
@@ -818,3 +818,137 @@ def test_assumptions(self, op, expr, assumptions, expected):
         assumptions = eval(assumptions)
         expected = eval(expected)
         assert evalrel(op, eqn, assumptions) == expected
+
+
+class TestComplexParts:
+    def setup_basic(self, dtype):
+        grid = Grid(shape=(5,), extent=(4.,))
+        f = Function(name='f', grid=grid, dtype=dtype)
+        f.data_with_halo[:] = np.arange(7) + 1j*np.arange(7, 14)[::-1]
+
+        f_real = Function(name='f_real', grid=grid)
+        f_imag = Function(name='f_imag', grid=grid)
+        return f, f_real, f_imag
+
+    def run_operator(self, eqs, language):
+        with switchconfig(language=language):
+            Operator(eqs)()
+
+    def test_devito_print(self):
+        f, _, _ = self.setup_basic(np.complex64)
+
+        assert str(Real(f)) == 'Real(f(x))'
+        assert str(Imag(f)) == 'Imag(f(x))'
+
+    @pytest.mark.parametrize('language', ['C', 'CXX', 'CXXopenmp'])
```

Comment:
If you just change the CI you won't need this parametrization and only check `configuration['language']` for the assert (and no need for switchconfig)